### DISCUSSION TITLE
Submission

### DISCUSSION THREAD

**Discussion by shayavh** [IBAHCM/RPiR#257] [Archive]
Hi @richardreeve @soniamitchell, could you just clarify how we hand in our participation for Wednesday? I know we shared our own repo in the discussions, but should we also share links to repos that we commented on in here? Thanks :)

**Comment by abaumane** [comment]
As far as I understood, on Moodle you submit your own repo link as well as a link to the post here where you and your roommates have shared their repo links. They will then see all the repos you commented on (since we should be doing that with all of our breakout roommates).

**Reply by richardreeve** [reply]
Yes, that's right. There should just be two links in the submission, to the repo and to the Registration discussion thread for your breakout room, unless you specfically commented on some other repos you want to draw our attention to I guess...
